R/ci.R

```diff
@@ -994,6 +994,19 @@ ci.meta_bma <- ci.meta_random
 
 
 
+# lmtest::coeftest -------------------------------------------------------
+
+#' @export
+ci.coeftest <- function(model, ci=0.95, ...) {
+  out <- stats::confint(model, level=ci)
+  out <- .data_frame(
+    Parameter = .remove_backticks_from_string(row.names(out)),
+    CI_low = out[, 1],
+    CI_high = out[, 2]
+  )
+  row.names(out) <- NULL
```

Comment:
You actually don't need to set rownames to NULL here, because this is done in `.data_frame()`.